CHANGELOG.rst

```diff
@@ -117,6 +117,7 @@ Fixed
   Contributed by Tatsuma Matsuki (@mtatsuma)
 
 * Fix dependency conflicts by updating ``requests`` (2.23.0) and ``gitpython`` (2.1.15). #4869
+* Fix orquesta syntax error for with items task where action is misindented or missing. (bug fix)
```

Comment:
Consider referencing the Orquesta PR here.

Reply:
Done!